lib/slack/real_time/stores/store.rb

```diff
@@ -236,21 +253,24 @@ def initialize(attrs)
         # @see https://api.slack.com/events/group_archive
         # @see https://github.com/slack-ruby/slack-api-ref/blob/master/events/group_archive.json
         on :group_archive do |data|
-          channel = groups[data.channel]
+          channel = private_channels[data.channel]
           channel.is_archived = true if channel
         end
 
         # You closed a private channel.
         # @see https://api.slack.com/events/group_close
         # @see https://github.com/slack-ruby/slack-api-ref/blob/master/events/group_close.json
         on :group_close do |data|
-          groups[data.channel].is_open = false
+          channel = private_channels[data.channel]
```

Comment:
Note that group events do say they are for private channels, but channel events do not say they are only for public ones (https://api.slack.com/events/channel_deleted).

Reply:
Yeah I noticed that. The Slack docs are incomplete in areas. I tested most of these events and observed only public channels triggering the `channel_` events while only private channels (both newer and group type) triggering the `group_` events.